### ISSUE TITLE
Cannot compile, installation fails

### ISSUE THREAD

**Issue by swarnendubiswas** [catppuccin/kde#6]
I am trying to build and install the package on Ubuntu 22.04, Plasma 5.25.5, but I get the following error. What are the dependencies to resolve the issue?

```
~❯ kpackagetool5 -i catppuccin.tar.gz
qt.network.ssl: QSslSocket: cannot resolve EVP_PKEY_param_check
qt.network.ssl: QSslSocket: cannot resolve OPENSSL_sk_new_null
qt.network.ssl: QSslSocket: cannot resolve OPENSSL_sk_push
qt.network.ssl: QSslSocket: cannot resolve OPENSSL_sk_free
qt.network.ssl: QSslSocket: cannot resolve OPENSSL_sk_num
qt.network.ssl: QSslSocket: cannot resolve OPENSSL_sk_pop_free
qt.network.ssl: QSslSocket: cannot resolve OPENSSL_sk_value
qt.network.ssl: QSslSocket: cannot resolve SSL_CTX_set_options
qt.network.ssl: QSslSocket: cannot resolve SSL_CTX_get_security_level
qt.network.ssl: QSslSocket: cannot resolve SSL_CTX_set_security_level
qt.network.ssl: QSslSocket: cannot resolve SSL_CTX_set_ciphersuites
qt.network.ssl: QSslSocket: cannot resolve SSL_set_psk_use_session_callback
qt.network.ssl: QSslSocket: cannot resolve SSL_SESSION_is_resumable
qt.network.ssl: QSslSocket: cannot resolve SSL_session_reused
qt.network.ssl: QSslSocket: cannot resolve SSL_set_options
qt.network.ssl: QSslSocket: cannot resolve DSA_bits
qt.network.ssl: QSslSocket: cannot resolve DTLSv1_listen
qt.network.ssl: QSslSocket: cannot resolve BIO_ADDR_new
qt.network.ssl: QSslSocket: cannot resolve BIO_ADDR_free
qt.network.ssl: QSslSocket: cannot resolve OCSP_resp_get0_certs
qt.network.ssl: QSslSocket: cannot resolve BN_is_word
qt.network.ssl: QSslSocket: cannot resolve SSL_CONF_CTX_new
qt.network.ssl: QSslSocket: cannot resolve SSL_CONF_CTX_free
qt.network.ssl: QSslSocket: cannot resolve SSL_CONF_CTX_set_ssl_ctx
qt.network.ssl: QSslSocket: cannot resolve SSL_CONF_CTX_set_flags
qt.network.ssl: QSslSocket: cannot resolve SSL_CONF_CTX_finish
qt.network.ssl: QSslSocket: cannot resolve SSL_CONF_cmd
qt.network.ssl: QSslSocket: cannot resolve SSL_get1_peer_certificate
qt.network.ssl: QSslSocket: cannot resolve EVP_PKEY_get_base_id
qt.network.ssl: QSslSocket: cannot resolve SSL_in_init
qt.network.ssl: QSslSocket: cannot resolve SSL_set_psk_client_callback
qt.network.ssl: QSslSocket: cannot resolve SSL_set_psk_server_callback
qt.network.ssl: QSslSocket: cannot resolve SSL_CTX_use_psk_identity_hint
qt.network.ssl: QSslSocket: cannot resolve SSL_CTX_load_verify_dir
qt.network.ssl: QSslSocket: cannot resolve DTLS_server_method
qt.network.ssl: QSslSocket: cannot resolve DTLS_client_method
qt.network.ssl: QSslSocket: cannot call unresolved function OPENSSL_sk_num
qt.network.ssl: QSslSocket: cannot call unresolved function DTLS_client_method
qt.network.ssl: QSslSocket: cannot call unresolved function SSL_CTX_get_security_level
qt.network.ssl: QSslSocket: cannot call unresolved function SSL_CTX_set_options
kf.newstuff.core: "Invalid or empty cipher list ()"
kns error: KNSCore::ProviderError "Loading of providers from file: https://autoconfig.kde.org/ocs/providers.xml failed" QVariant(QString, "https://autoconfig.kde.org/ocs/providers.xml")
Error: Installation of /home/swarnendu/github/kde/kde-store-archives/global-theme/catppuccin.tar.gz failed: Could not install dependency: 'kns://colorschemes.knsrc/api.kde-look.org/1690362'

```

**Comment by Sourcastic** [comment]
Hmm, it seems to me like it couldn't download the Catppuccin colour scheme from the KDE Store. I;m not sure what the solution could be, but retrying could be worth a shot. To do that, delete the Catppuccin folder from .local/Plasma/look-and-feel and run the command again. Alternatively, go to the colour schemes section in settings and try to download the colour scheme separately from the get new button.

**Comment by swarnendubiswas** [comment]
System Settings crashes for me when I try to install/get new global themes. Is that why the installation is failing? I thought there is an issue in compiling source files.

**Comment by Sourcastic** [comment]
Might be the case. Happens to me too. It fixes itself later magically.

**Comment by Sourcastic** [comment]
So, just to elaborate, this issue isn't with catppuccin, people are reporting it on a lot of distros. Seems to be some kind of backend problem. [As an example.](https://www.reddit.com/r/kde/comments/xcho7r/debian_115_issues_with_ssl_handshake_failed/)

**Comment by nekowinston** [comment]
Should be fixed with 69dc8315057cee375dfa5374ced74cb95df99d71
